### PR TITLE
Templates' design: organizing pages and creating links for all needed…

### PR DESCRIPTION
**Pull request by nikolina-p**
… functionalities.
@kunicmarko20

### CODE REVIEW

File: templates/base.html.twig [hunk 1]
```diff
@@ -4,10 +4,39 @@
...
-        {% block javascripts %}{% endblock %}
+        <div class="header">
+            {% block header %}
+                <img class="header" src="{{ asset("photos/header.png") }}">
+            {% endblock %}
+        </div>
+        <div id="navigation">
+            {% block navigation %}
+                {% if is_granted('ROLE_USER') or is_granted('ROLE_ADMIN') %}
+                    <span><a href="/all">logout</a></span>
```
Comment (kunicmarko20) [inline templates/base.html.twig]: logout going to `/all`?

File: templates/user/user.html.twig [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {% extends "base.html.twig" %}
 
+{% block sidebar %}
```
Comment (kunicmarko20) [inline templates/user/user.html.twig]: This is the 3rd time I see the same block, it should be in base file

File: templates/base.html.twig (hunk 1 above)
Comment (nikolina-p) [reply templates/base.html.twig]: that is just for now... logout is a task that is not done yet.

File: templates/base.html.twig [hunk 3]
```diff
@@ -4,10 +4,43 @@
...
-        {% block javascripts %}{% endblock %}
+        <div class="header">
+            {% block header %}
+                <img class="header" src="{{ asset("photos/header.png") }}">
+            {% endblock %}
+        </div>
+        <div id="navigation">
+            {% block navigation %}
+                {% if is_granted('ROLE_USER') or is_granted('ROLE_ADMIN') %}
+                    <span><a href="/logout">logout</a></span>
```
Comment (kunicmarko20) [inline templates/base.html.twig]: You should be using `paths` https://symfony.com/doc/current/templating.html#linking-to-pages for showing the url

**Pull request merged**